datastore/google/cloud/datastore/__init__.py

```diff
@@ -14,38 +14,40 @@
 
 """Shortcut methods for getting set up with Google Cloud Datastore.
 
-You'll typically use these to get started with the API::
+You'll typically use these to get started with the API:
 
-    >>> from google.cloud import datastore
-    >>>
-    >>> client = datastore.Client()
-    >>> key = client.key('EntityKind', 1234)
-    >>> entity = datastore.Entity(key)
-    >>> query = client.query(kind='EntityKind')
+.. code-block:: python
+
+  >>> from google.cloud import datastore
+  >>>
+  >>> client = datastore.Client()
+  >>> key = client.key('EntityKind', 1234)
+  >>> entity = datastore.Entity(key)
+  >>> query = client.query(kind='EntityKind')
 
```

Comment:
I'm happy to change them all back to the explicit block `::`. I see this as a temporary band-aid anyhow, since I will be introducing a check that all literal blocks are valid doctests (or bash or something else)